src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,44 @@
+.navbar {
+  border-bottom: 1px solid map-get($pp-colours, "grey-lighten-3");
+  padding: 15px;
+  box-shadow: 0px 3px 4px 0px rgba(0, 0, 0, 0.1);
```

Comment:
We have to decide on box shadow for nav-bar and footer

Reply:
Once they're both merged i'll create a variable & update it for them to match